NetSDK/Services/Client/Classes/SelfRefreshingClient.cs

```diff
@@ -165,22 +164,34 @@ public void Stop()
 
         private void InitializeLogger()
         {
-            Hierarchy hierarchy = (Hierarchy)LogManager.GetRepository();
-            if (hierarchy.Root.Appenders.Count == 0)
-            {
-                RollingFileAppender fileAppender = new RollingFileAppender();
-                fileAppender.AppendToFile = true;
-                fileAppender.LockingModel = new FileAppender.MinimalLock();
-                fileAppender.File = @"Logs\split-sdk.log";
-                fileAppender.RollingStyle = RollingFileAppender.RollingMode.Date;
-                fileAppender.DatePattern = "yyyyMMdd";
-                PatternLayout pl = new PatternLayout();
-                pl.ConversionPattern = "%date %level %logger - %message%newline";
-                pl.ActivateOptions();
-                fileAppender.Layout = pl;
-                fileAppender.ActivateOptions();
+            var fileTarget = new FileTarget();
```

Comment:
I configure the logger for each Split Client child class. If you prefer, I can move the configuration of the logger, to the factory class, as I did for Net core